src/assets/languages/english.json

```diff
@@ -67,6 +67,29 @@
         "registrationFail": "Registration unsuccesful"
       }
     },
+    "userUpdate": {
+      "messages": {
+        "heading": "Update your profile!",
+        "buttonMessage": "UPDATE",
+        "hoverProfile": "Click to upload",
```

Comment:
Change 'Click to upload' to "Change profile picture", and on clicking a dialog appears asking the user if they want to upload or remove the picture.

Reply:
@Raxerz "first upload" message would serve as a default message when he is not having the profile picture already. For the add/delete functionality, another dialog would need to be implemented. Is it a priority?

Reply:
@shreyans29 @paavininanda @Buddhiprabha Should we take it in this PR or should we create a separate issue for it?

Reply:
Yes I too think it is required. If this was not there in the original suggestions, shall we add it as another issue and assign to @lunayach? @Raxerz

Reply:
@Buddhiprabha Yes lets add it as another issue.

Reply:
Oops ! late to the discussion, but I would also suggest another Issue.